blog.go

```diff
@@ -38,6 +38,7 @@ type Blog struct {
 	TemplateSuffix     string     `json:"template_suffix"`
 	CreatedAt          *time.Time `json:"created_at"`
 	UpdatedAt          *time.Time `json:"updated_at"`
+	AdminGraphqlAPIID  string     `json:"admin_graphql_api_id,omitempty"`
```

Comment:
As per Golang convention, can you change the capitalization to `AdminGraphqlApiId` please?